test/taproot-check-test.js

```diff
@@ -68,6 +69,22 @@ describe('Taproot Check', function() {
       );
     });
 
+    it('should have invalid schnorr signature size', () => {
+      const mtx = new MTX();
+      mtx.outputs.push(new Output({ value: 1e8 - 10000 }));
+      mtx.addCoin(keyspendUTXO);
+
+      const script = new Script();
+      script.pushOp(opcodes.OP_CHECKSIGVERIFY);
+
+      mtx.inputs[0].witness.push(script.toRaw());
```

Comment:
I'd prefer if the test proved the most minimal violation of a rule. So instead of just completely replacing the signature in a key path spend with what is essentially just a random byte, let's start with a valid signed tx (we proved this method is valid earlier in the test) and then lop off one byte from the end of the signature to make it too short:

```suggestion
      signMTX(mtx);
      mtx.inputs[0].witness.items[0] = mtx.inputs[0].witness.items[0].slice(0, -1);
```

Reply:
Yup. I have made the changes. Thanks